package.json

```diff
@@ -22,6 +22,7 @@
         "build-prod:data-atlas": "ng build --configuration production data-atlas-app",
         "build-prod:ebv-analyzer": "ng build --configuration production ebv-analyzer-app",
         "build-prod:geoengine": "ng build --configuration production geoengine-app",
+        "build-watch:gfbio": "ng build --configuration development gfbio-app --watch",
```

Comment:
Why do you need build-watch and not just use `serve:gfbio:local"`?

Reply:
Sorry, my bad: aec4bc46d8bd909c677177abe4662c06a34cb437